extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * .amp-lightboxed gets added to the element when displayed in the lightbox.
+ * .amp-lightboxed is designed to provide a reasonable default behaviour for
+ * lightboxing of arbitary elements.
+ * Both components and page authors are free to customize the style of the
+ * element when inside the lightbox.
+ * Example:
+ *    .myDiv {
+ *       color: blue;
+ *     }
+ *
+ *     .myDiv.amp-lightboxed {
+ *       color: red;
+ *     }
+ */
+.amp-lightboxed {
+  position: fixed !important;
+  z-index: 2147483645 !important;
+  width: 100%;
```

Comment:
`object-fit: contain` makes sense to be part of the default behaviour. added it.